Error while running `npm run server` inside the Faucet example folder

**Describe the bug**
I get an error while trying to execute `npm run server` inside the `\js-chain-libs\examples\faucet` folder. 

**To Reproduce**
Steps to reproduce the behavior:
1. go to `\js-chain-libs\examples\faucet` folder
2. execute `cd ../../ && wasm-pack build --target nodejs --out-dir ./examples/faucet/js-chain-libs/`
3. execute `npm install`
4. go back to `\js-chain-libs\examples\faucet` and execute `npm run server`

**Expected behavior**
No error should be received. The server should be successfully started.
Or clearer instructions should be provided.

**Additional context**
```
D:\iohk\otherProjects\js-chain-libs\examples\faucet>npm run server

> faucet@1.0.0 server D:\iohk\otherProjects\js-chain-libs\examples\faucet
> node server.js

internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'fastify'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (D:\iohk\otherProjects\js-chain-libs\examples\faucet\server.js:3:17)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! faucet@1.0.0 server: `node server.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the faucet@1.0.0 server script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Dorin\AppData\Roaming\npm-cache\_logs\2019-08-13T08_28_28_021Z-debug.log
```

[2019-08-13T08_28_28_021Z-debug.log](https://github.com/input-output-hk/js-chain-libs/files/3496018/2019-08-13T08_28_28_021Z-debug.log)

Ok. I did the new sequence of steps but I still get an error.
I am using a `Windows 10` machine.

**Steps**:
1. go to `\js-chain-libs' folder/root
2. execute `wasm-pack build --target nodejs --out-dir ./examples/faucet/js-chain-libs/`
3. go to `\js-chain-libs\examples\faucet` folder
4. execute 'npm install'
5. execute `npm run server`

**Result**:
```
D:\iohk\otherProjects\js-chain-libs>wasm-pack build --target nodejs --out-dir ./examples/faucet/js-chain-libs/
[INFO]: Checking for the Wasm target...
[INFO]: Compiling to Wasm...
    Finished release [optimized] target(s) in 1.53s
[INFO]: Installing wasm-bindgen...
[INFO]: Optional fields missing from Cargo.toml: 'description', 'repository', and 'license'. These are not necessary, but recommended
[INFO]: :-) Done in 3.01s
[INFO]: :-) Your wasm pkg is ready to publish at .\./examples/faucet/js-chain-libs/.
```
```
D:\iohk\otherProjects\js-chain-libs>cd examples\faucet
```
```
D:\iohk\otherProjects\js-chain-libs\examples\faucet>npm install
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN faucet@1.0.0 No repository field.

added 52 packages from 41 contributors and audited 78 packages in 4.351s
found 0 vulnerabilities
```

```
D:\iohk\otherProjects\js-chain-libs\examples\faucet>npm run server

> faucet@1.0.0 server D:\iohk\otherProjects\js-chain-libs\examples\faucet
> node server.js

{"level":50,"time":1565773088249,"pid":6064,"hostname":"lenovo","errors":[{"keyword":"required","dataPath":"","schemaPath":"#/required","params":{"missingProperty":"SECRET_KE
Y"},"message":"should have required property 'SECRET_KEY'"}],"msg":"should have required property 'SECRET_KEY'","stack":"Error: should have required property 'SECRET_KEY'\n
  at loadAndValidateEnvironment (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\env-schema\\index.js:59:19)\n    at loadAndValidateEnvironment (D:\\i
ohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\fastify-env\\index.js:8:20)\n    at Plugin.exec (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\
node_modules\\avvio\\plugin.js:89:17)\n    at Boot.loadPlugin (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\avvio\\plugin.js:175:10)\n    at Task.r
elease (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\fastq\\queue.js:127:16)\n    at worked (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\fauc
et\\node_modules\\fastq\\queue.js:169:10)\n    at toLoad.finish (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\avvio\\plugin.js:178:7)\n    at done
(D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\avvio\\plugin.js:136:5)\n    at check (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node
_modules\\avvio\\plugin.js:147:7)\n    at process._tickCallback (internal/process/next_tick.js:61:11)","type":"Error","v":1}
{"level":50,"time":1565773088252,"pid":6064,"hostname":"lenovo","msg":"Cannot read property 'PORT' of undefined","stack":"TypeError: Cannot read property 'PORT' of undefined\
n    at start (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\server.js:145:41)\n    at fastify.register.ready.err (D:\\iohk\\otherProjects\\js-chain-libs\\example
s\\faucet\\server.js:140:3)\n    at Object._encapsulateThreeParam (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\avvio\\boot.js:408:13)\n    at Boot
.callWithCbOrNextTick (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\avvio\\boot.js:339:5)\n    at release (D:\\iohk\\otherProjects\\js-chain-libs\\
examples\\faucet\\node_modules\\fastq\\queue.js:127:16)\n    at Object.resume (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\fastq\\queue.js:61:7)\n
    at Boot.Plugin.loadPlugin.call (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\avvio\\boot.js:155:20)\n    at toLoad.finish (D:\\iohk\\otherProje
cts\\js-chain-libs\\examples\\faucet\\node_modules\\avvio\\plugin.js:178:7)\n    at done (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\avvio\\plugi
n.js:136:5)\n    at check (D:\\iohk\\otherProjects\\js-chain-libs\\examples\\faucet\\node_modules\\avvio\\plugin.js:147:7)","type":"Error","v":1}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! faucet@1.0.0 server: `node server.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the faucet@1.0.0 server script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Dorin\AppData\Roaming\npm-cache\_logs\2019-08-14T08_58_08_279Z-debug.log
```

--- a/examples/faucet/README.md
+++ b/examples/faucet/README.md
@@ -6,14 +6,15 @@ A minimal example of a faucet endpoint, using the sdk on nodejs, showcasing its 
 
 ### Install dependencies
 
-1. Build the sdk for nodejs target, in the root directory of this example run:
+1. Build the sdk for nodejs target, in the root directory of this repository run:
 
 ```sh
-cd ../../ && wasm-pack build --target nodejs --out-dir ./examples/faucet/js-chain-libs/
+wasm-pack build --target nodejs --out-dir ./examples/faucet/js-chain-libs/
 ```
 
-2. Install the other node dependencies: `npm install`
+2. Install the example node dependencies:
 
+In the root directory of the faucet example run: `npm install`
 
 ### Configuration
 

